examples/secureTokenAndroidKeyStore/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:24.2.0'
     testCompile 'junit:junit:4.12'
+    compile 'io.realm:secure-userstore:1.0.0'
```

Comment:
do we need an `android` in the lib name like `io.realm:android-secure-userstore`